app/views/govuk_publishing_components/components/_select.html.erb

```diff
@@ -3,6 +3,7 @@
   id ||= false
   label ||= false
   full_width ||= false
+  name_attr ||= id
```

Comment:
I think we use `name` for this for most of the components

Reply:
Oh, do we? I typed in `name` and my editor highlighted it, so I assumed it was reserved.

Reply:
Ah yes that works. Will change.